lib/vis.py

```diff
@@ -254,8 +254,8 @@ def generate_log_plots(filter_val, plot_data, output_directory, output_file_name
         sum_each_row.append(row)
 
     # print sum_each_row
```

Comment:
@kaushik-rohit This comment of sum_each_row is misleading. The comment may be intended for line-254, in which case it would either come between lines-253 and 254 or at the end of line-254.